spec/Section 3 -- Type System.md

```diff
@@ -354,35 +388,35 @@ represent additional fields a GraphQL client only accesses locally.
 
 ScalarTypeDefinition : Description? scalar Name Directives[Const]?
 
-Scalar types represent primitive leaf values in a GraphQL type system. GraphQL
-responses take the form of a hierarchical tree; the leaves of this tree are
-typically GraphQL Scalar types (but may also be Enum types or {null} values).
+:: A *Scalar type* represents primitive leaf values in a GraphQL type system.
+GraphQL responses take the form of a hierarchical tree; the leaves of this tree
+are typically GraphQL Scalar types (but may also be Enum types or {null} values).
```

Comment:
When we're talking about `types` I'm not sure it makes sense to include `{null}`? This might imply you could have `type A {foo: null}`. We should also factor in that the leaves may be lists/non-nulls:

```suggestion
:: A *Scalar type* represents primitive leaf values in a GraphQL type system.
GraphQL responses take the form of a hierarchical tree; the *named type* of the
leaves of this tree are typically GraphQL Scalar types (but may also be Enum
types).
```

Reply:
Agreed. This was a case where we were mixing "types" and "values". I'll rewrite this